test_util/test_aws_cf_launch_test.py

```diff
@@ -0,0 +1,50 @@
+"""WARNING: DO NOT MERGE THIS FILE
```

Comment:
No, this file is not to be included as intended with all my warnings. The only reason this is in python and not pure bash is that I wanted to hack into the current CI commands to get a status check on this PR. We do not want these sloppy runner scripts. That's the whole point of dcos launch. We do not want to be creating a new interface everytime we want to make a marginally different new test. One of the biggest motivations of making dcos-launch is that we can just plug this into our CI:
```
#!/bin/bash
wget $DCOS_LAUNCH_URL
set -o errexit
cat <<EOF > config.yaml
---
this_is_a_temporary_config_format_do_not_put_in_production: yes_i_agree
type: cloudformation
template_url: $template_url
stack_name: $stack_name
provider_info:
  region: us-west-2
  access_key_id: $access_key_id
  secret_access_key: $secret_access_key
ssh_info:
  user: core
parameters:
  - ParameterKey: AdminLocation
    ParameterValue: 0.0.0.0/0
  - ParameterKey: PublicSlaveInstanceCount
    ParameterValue: 1
  - ParameterKey: SlaveInstanceCount
    ParameterValue: 2
EOF
./dcos-launch create --config-path=config.yaml
./dcos-launch wait
./dcos-launch describe
./dcos-launch test -- py.test -s -rs -vv --teamcity
./dcos-launch delete
```
In this code block, you are _only_ dependent upon being able to download one executable and every necessary setting to control this test is listed in one place. You can replace any part of this with parameters at anytime without committing code changes.

If we keep this file that I say must go, the testing looks like this:
```
# get artifacts from previous test, which is TC specific
# create and activate a new virtual environment
# install everything in that virtual environment
# read in a bunch of envvars that are set elsewhere in CI
test-aws-cf-launch
```
Now we need completely rebuild our software just to test a different configuration (e.g. 4 slaves)